packages/tm-bridge/CHANGELOG.md

```
@@ -6,3 +6,10 @@
 
 - Updated dependencies []:
   - @tm/core@null
+
+## null
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)